tests/unit/services/service-worker-update-test.js

```diff
@@ -0,0 +1,28 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import {
+  setupServiceWorkerUpdater,
+  serviceWorkerUpdate,
+} from 'ember-service-worker-update-notify/test-support/updater';
+
+module('Unit | Service | service-worker-update', function(hooks) {
```

Comment:
These tests are tiny / amazing :bowing_man:

Reply:
should this test include a polling test?

like, the `waitMs` a time or two, invoke serviceWorkerUpdate, and check if the service `hasUpdate`?

Reply:
though, I guess the `await` part of serviceWorkerUpdate covers that. hmm

Reply:
in a follow-up PR, it'd be great to hook up code coverage. :man_shrugging: